qartod/metbk/metbk_qartod_gross_range_test_values.csv

```diff
@@ -1,25 +1,89 @@
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'longwave_irradiance'},"{'qartod': {'gross_range_test': {'suspect_span': [237.3, 395.5], 'fail_span': [0.0, 700.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'precipitation'},"{'qartod': {'gross_range_test': {'suspect_span': [0.64045, 49.79], 'fail_span': [0.0, 50.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'shortwave_irradiance'},"{'qartod': {'gross_range_test': {'suspect_span': [0.0, 997.75751], 'fail_span': [0.0, 2800.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'sea_surface_temperature'},"{'qartod': {'gross_range_test': {'suspect_span': [8.38996, 17.37216], 'fail_span': [-5.0, 45.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'sea_surface_conductivity'},"{'qartod': {'gross_range_test': {'suspect_span': [2.96245, 4.13711], 'fail_span': [0.0, 7.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'met_salsurf'},"{'qartod': {'gross_range_test': {'suspect_span': [26.02579, 33.54789], 'fail_span': [0.0, 42.0]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
+CE02SHSM,SBD11,06-METBKA000,metbk_a_dcl_instrument,{'inp': 'eastward_wind_velocity'},"{'qartod': {'gross_range_test': {'suspect_span': [-9.92, 9.77], 'fail_span': [-32.5, 32.5]}}}",User range based on data collected through 2022-01-01.,"User range based on percentiles of the observations, which are not normally distributed. Percentiles were chosen to cover 99.7% of the data, approximating the Empirical Rule."
```

Comment:
Is the gross range sensor range here correct? The manual for the WindObserver II on the ASIMET say 0-65 m/s, which using minimum values for u and v would be 45 m/s for both vectors.